riscv/csrs.cc

```diff
@@ -607,7 +607,6 @@ misa_csr_t::misa_csr_t(processor_t* const proc, const reg_t addr, const reg_t ma
                         | (1L << ('F' - 'A'))
                         | (1L << ('D' - 'A'))
                         | (1L << ('Q' - 'A'))
-                        | (1L << ('C' - 'A'))
```

Comment:
The comment a few lines up needs updating too

Reply:
Ack.